make_test_images/src/make_test_images.rs

```diff
@@ -36,6 +37,7 @@ fn get_signer_with_alg(alg: &str) -> c2pa::Result<Box<dyn Signer>> {
     signcert_path.push(format!("../sdk/tests/fixtures/certs/{}.pub", alg));
     let mut pkey_path = PathBuf::from(env!("CARGO_MANIFEST_DIR"));
     pkey_path.push(format!("../sdk/tests/fixtures/certs/{}.pem", alg));
+    let alg: SigningAlg = alg.parse().map_err(|_| c2pa::Error::UnsupportedType)?;
```

Comment:
I was wondering how this would work. We will need to do the same in c2patool. I'm presuming that due to the string mapping, when we convert one of these values to json, we just get the string representation. It needs to round trip well in serde.

Reply:
If you have a SigningAlg instance, you can call `to_string()` on it and it will round-trip. Off the top of my head (not actually tested), this should work:

```rust
let alg = SigningAlg::Ps256;
assert_eq!(alg.to_string(), "ps256");
```